src/scripts/build_synthetic_nts.py

```diff
@@ -0,0 +1,87 @@
+"""Script that builds a synthetic vocie audio from reading 
+the nst dataset. 
+
+Usage:
+    python build_syntethic_nts.py 
+"""
+
+import os
+import subprocess
+
+from gtts import gTTS
+import pandas as pd
+
+
+def generate_speech_mac(text, filename):
+    """Generate speech from text using macOS 'say' command and save it to a file.
+    The downloaded voices needs to be changed manually in 
+    spoken content in settings of the mac machine.
+    The file is saved in a special mac sound format called ".aiff".
+    Extra details see: 
+    "https://maithegeek.medium.com/having-fun-in-macos-with-say-command-d4a0d3319668"
+
+    Args:
+        text (str): The text to convert to speech.
+        filename (str): The name of the output audio file.
+
+    Returns:
+        None
+    """
```

Comment:
```suggestion
    """Generate speech from text using macOS 'say' command and save it to a file.
    
    The downloaded voices needs to be changed manually in 
    spoken content in settings of the mac machine.
    The file is saved in a special mac sound format called ".aiff".
    Extra details see: 
    "https://maithegeek.medium.com/having-fun-in-macos-with-say-command-d4a0d3319668"

    Args:
        text: The text to convert to speech.
        filename: The name of the output audio file.
    """
```

Reply:
I have added the licens problem. Only for private use.

Reply:
I have made the space.